source/_integrations/eq3btsmart.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: eQ-3 Bluetooth Smart Thermostats
+description: Instructions on how to integrate eQ-3 Bluetooth Smart Thermostats into Home Assistant.
+ha_category: Climate
+ha_iot_class: Local Polling
+ha_release: 0.18
```

Comment:
not sure if we should bump this to 2024.3 since its technically new again

Reply:
Yeah, I agree, I'll change it to 2024.3.